### ISSUE TITLE
Impossible to Install

### ISSUE THREAD

**Issue by stringcgg** [asciidisco/plugin.video.netflix#105]
Why not make this easier to install?

**Comment by asciidisco** [comment]
I wouldn't say "impossible" as I know quite a few people who managed to do so.
Open to suggestions on how to make this "easier".
What was the part you found hardest or you couldn't cope with?

**Comment by skarragallagher** [comment]
I struggled with the install as well. It certainly isn't impossible, but took quite a bit of digging find the issue. Initially the problem was two fold, Instructions are somewhat outdated, and two Kodi X64 recent versions have a broken version of one of the dependencies. Try installing 32bit version from 9/8/2017. Also the widevine architecture needs to match. For instance a 64 bit version of chrome seems to contain a 64bit version of the DLL. Once I got the 32bit version of kodi from 9/8 and a 32 bit version of the widevine everything worked properly.

Also the instructions seem to reference a german site for installing widevine. I had to go through google translate to figure this out only to realize that half of the process is not needed (symlinking)

I do realize this is a work in progress though and obviously bugs will be worked out with Kodi as development continues.
